models/preles/R/runPRELES.jobsh.R

```diff
@@ -42,40 +40,85 @@ runPRELES.jobsh<- function(met.file,outdir,start.date,end.date){
+  Precip <-ncvar_get(nc,"precipitation_flux")#precipitation in kg/m2s1
   CO2 <-ncvar_get(nc,"mole_fraction_of_carbon_dioxide_in_air") #mol/mol
-  
+  RH <- ncvar_get(nc,"relative_humidity")
   lat<-ncvar_get(nc,"latitude")
   lon<-ncvar_get(nc,"longitude")
   
+  ## GET VPD Temperature and realtive humidity
+  VPD = get.vpd(RH,Tair)
+  VPD = VPD * .01 # convert to Pascal
+  
   ## Format/convert inputs 
   PAR= tapply(PAR, doy,mean,na.rm=TRUE) #Find the mean for the day
-  TAir=ud.convert(tapply(Tair,doy,mean,na.rm=TRUE),"K", "C")#Convert Kelvin to Celcius
+  TAir=ud.convert(tapply(Tair,doy,mean,na.rm=TRUE),"kelvin", "celsius")#Convert Kelvin to Celcius
   VPD= ud.convert(tapply(VPD,doy,mean,na.rm=TRUE), "Pa","kPa")#pascal to kila pascal
   Precip=tapply(Precip,doy,sum, na.rm=TRUE) #Sum to daily precipitation
-  CO2= ud.convert(tapply(CO2,doy,sum),"mol/mol","ppm") #need daily average so sum up day
+  CO2= tapply(CO2,doy,mean) #need daily average, so sum up day
+  CO2= CO2/1e6
   doy=tapply(doy,doy,mean) # day of year
-  fAPAR =rep (0.8,length=length(doy)) #For now set to 0.8. Needs to be between 0-1
+  fAPAR =rep (0.6,length=length(doy)) #For now set to 0.6. Needs to be between 0-1
   
   ##Bind inputs 
   tmp<-cbind (PAR,TAir,VPD,Precip,CO2,fAPAR)
+  param.table = c(413.0, ## 1 soildepth |
+                  0.450, ## 2 ThetaFC
+                  0.118, ## 3 ThetaPWP
+                  3, ## 4 tauDrainage
+                  ## GPP_MODEL_PARAMETERS
+                  0.748018, ## 5 betaGPP
+                  13.23383, ## 6 tauGPP
+                  -3.9657867, ## 7 S0GPP
+                  18.76696, ## 8 SmaxGPP
+                  -0.130473, ## 9 kappaGPP
+                  0.034459, ## 10 gammaGPP
+                  0.450828, ## 11 soilthresGPP
+                  2000, ## 12 cmCO2
+                  0.4, ## 13 ckappaCO2
+                  ## EVAPOTRANSPIRATION_PARAMETERS
+                  0.324463, ## 14 betaET
+                  0.874151, ## 15 kappaET
+                  0.075601, ## 16 chiET
+                  0.541605, ## 17 soilthresET
+                  0.273584, ## 18 nu ET
+                  ## SNOW_RAIN_PARAMETERS
+                  1.2, ## 19 Meltcoef
+                  0.33, ## 20 I_0
+                  4.970496, ## 21 CWmax, i.e. max canopy water
+                  0, ## 22 SnowThreshold, 
+                  0, ## 23 T_0, 
+                  200, ## 24 SWinit, ## START INITIALISATION PARAMETERS 
+                  0, ## 25 CWinit, ## Canopy water
+                  0, ## 26 SOGinit, ## Snow on Ground 
+                  20, ## 27 Sinit ##CWmax
+                  -999, ## t0 fPheno_start_date_Tsum_accumulation; conif -999, for birch 57
+                  -999, ## tcrit, fPheno_start_date_Tsum_Tthreshold, 1.5 birch
+                  -999 ##tsumcrit, fPheno_budburst_Tsum, 134 birch
+  )
   
+  ## Replace defualt with priors
+  params=read.table(priors)
```

Comment:
referring to parameters by number here is very fragile -- if the PFT is changed at all you'll get nonsensical and hard to debug results. Much more robust to do parameter matching by name.